deckhouse-controller/pkg/controller/module-controllers/source/controller.go

```diff
@@ -424,8 +466,13 @@ func (c *Controller) createModuleRelease(ctx context.Context, ms *v1alpha1.Modul
 			APIVersion: "deckhouse.io/v1alpha1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   fmt.Sprintf("%s-%s", moduleName, result.ModuleVersion),
-			Labels: map[string]string{"module": moduleName, "source": ms.Name, "release-checksum": checksum},
+			Name: fmt.Sprintf("%s-%s", moduleName, result.ModuleVersion),
+			Labels: map[string]string{
+				"module":               moduleName,
+				"source":               ms.Name,
+				"release-checksum":     checksum,
+				"module-update-policy": policyName,
```

Comment:
Maybe we can make this label as `update-policy`, to make it a bit shorter

Reply:
done

Reply:
was renamed to modules.deckhouse.io/updaet-policy :carpentry_saw: